### PR TITLE
Implemented MTL conversion for ACES and whatnot

### PR DESCRIPTION
**Pull request by StandingPadAnimations**
ACES and other colorspaces like it have issues when it comes to OBJ importing, so we need to convert the MTL so that Blender doesn't throw an exception when importing.

This comments out lines starting `map_d` from the MTL. The operation itself shouldn't affect MCprep since we don't deal with it anyway.

### CODE REVIEW

**Comment by StandingPadAnimations** [comment]
There is one small thing I need to implement as well: caching

Right now this branch will always copy the MTL to the backup folder, but it doesn't convert the MTL back to its original form. I'm thinking of adding a small "header" at the end of the edited MTL that says the following:
```
# This section was added by MCprep's MTL conversion script in order to allow importing without an
# error being thrown
# <timestamp of copied MTL>
```

**Comment by StandingPadAnimations** [comment]
Just adding this for tommorow me:
- Add MTL recovering (since we can reverse the operations MCprep does to convert MTLs) 
- Simplify header checking (inatead of list comprehension, we should just check the last 3 lines since MCprep will always write the header as the last part of the MTL)

**Review by TheDuckCow — CHANGES_REQUESTED** [review]
Sorry for all the nit picks, but it's getting there.

As I said, let me know if you have any interest in making the test script, otherwise I can do it.

File: MCprep_addon/world_tools.py [hunk 1]
```diff
@@ -87,6 +87,64 @@ def detect_world_exporter(filepath):
 	return 'jmc2obj'
 
 
+def convert_mtl(filepath):
```
[the remark this answers is not in the document]
Comment (StandingPadAnimations) [reply MCprep_addon/world_tools.py:90]: MTL conversion simply does the following:
- Comment out lines that begin with `map_d`
- Add a header at the end 

Here's an [original MTL](https://cdn.discordapp.com/attachments/819070016036667412/1083793955071729754/ATLAS.mtl) and the [edited MTL](https://cdn.discordapp.com/attachments/819070016036667412/1083793984448643174/ATLAS.mtl)

**Comment by StandingPadAnimations** [comment]
@TheDuckCow Made the necessary changes, although some guarantees may need to be added (like will `self.filepath` always be absolute?). Other then that, I think this is for the most part done

**Comment by TheDuckCow** [comment]
FYI @StandingPadAnimations of a minor merge conflict resolved, in case you do further work that you'll need to pull first. Reviewing shortly!

File: MCprep_addon/world_tools.py [hunk 2]
```diff
@@ -142,6 +143,83 @@ def detect_world_exporter(filepath):
...
+		"Raw",
+		"False Color"
+	)
+	mtl = filepath.rsplit(".", 1)[0] + '.mtl'
+	lines = None
+	copied_file = None
+	with open(mtl, 'r') as mtl_file:
+		lines = mtl_file.readlines()
+
+	if bpy.context.scene.view_settings.view_transform not in blender_standard:
```
Comment (TheDuckCow) [inline MCprep_addon/world_tools.py:170]: FYI only in my branched, I inverted this logic and returned early, cutting out the amount of indention below.

There it also returns "None" to indicate no conversion happened, which is a useful state imo.

**Review by TheDuckCow — APPROVED** [review]
Approving this.

Ready to merge, and then I'd move on to look at my PR followup [here](https://github.com/TheDuckCow/MCprep/pull/388) (don't review until this PR has been merged though; I couldn't use yours as a base since it's a different repo).

Also, is it fair to say that this change is actually somewhat negated now that we are moving to the new C++ obj importer in blender 3.5? I realize it'll still be helpful for the 3.4 and older users, but I just wanted to make sure I am aware.

**Comment by StandingPadAnimations** [comment]
> Also, is it fair to say that this change is actually somewhat negated now that we are moving to the new C++ obj importer in blender 3.5? I realize it'll still be helpful for the 3.4 and older users, but I just wanted to make sure I am aware.

From what I can tell, the behavior of the new C++ importer is either an oversight or now it checks for the existance of whatever colorspace it wants to use. As such, it's also good to fix the root of the issue, just in case

**Pull request merged**